canonical-web-and-design/certification.ubuntu.com
# No need for sass in this project

This project just contains the static CSS copied from the old certification.ubuntu.com site.

When this is redesigned, it will be moved to ubuntu.com/certification, and so be inside
the ubuntu.com codebase with all its Sass pipeline etc.

So we don't need the Sass stuff in this project.

QA
--

``` bash
$ ./run clean
...
$ ./run build
...
Nothing to build
$ ./run watch
...
Nothing to watch
$ ./run serve
...
2019-08-15 12:29:29.894Z INFO gunicorn.error "Listening at: http://0.0.0.0:8034 (19)"
```

Go to http://0.0.0.0:8034, check the site's styling still looks fine.
